types/svg-path-contours/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please remove this file and fix all the errors

Reply:
If I remove this then in the `index.d.ts` on line `type ClosePathCommand = ["z" | "Z"];` eslint shows an error `Type [T] is a single-element tuple type. You probably meant T[]`

Reply:
you can disable it for that line but not for whole project

Reply:
okay got it! Thanks